pernstrong/beers
# Bugfix/features saving

#### What's this PR do?
This is more or less the MVP for functionality. There are some bugs and a lot of styling to do. Also has a good amount of unit tests.
#### Where should the reviewer start?
App.tsx
#### How should this be manually tested?
Search for a brewery, add to favorites, play around with notes section and refresh to see if things save.
#### Any background context you want to provide?
n/a
#### What are the relevant tickets?
n/a
#### Screenshots (if appropriate)
n/a
#### Questions:
n/a
